imod/mf6/simulation.py

```diff
@@ -87,6 +87,22 @@ def get_packages(simulation: Modflow6Simulation) -> dict[str, Package]:
     }
 
 
+def validate_version(version_saved: str) -> None:
+    if version_saved and version_saved["imod-python"] != imod.__version__:
+        warnings.warn(
+            "iMOD Python version in current environment different from version with which simulation was dumped."
```

Comment:
The iMOD Python version in the current environment is different from the version with which the simulation was dumped.